utm-domains/utm-domain-commons.yaml

```diff
@@ -1101,7 +1103,9 @@ definitions:
         format: double
         description: >-
           The direction of travel relative to track_bearing_reference in
-          degrees. Value must be >= 0.0 and < 360.0.  Note that FIXM allows for
+          degrees.
+          track_bearing value MUST be >= 0.0 and < 360.0.
```

Comment:
Let's put empty line above and below the inequality line to emphasize.  Perhaps same with other inequalities above.